lang/gcc11/Portfile

```diff
@@ -7,13 +7,13 @@ PortGroup           active_variants              1.1
 PortGroup           conflicts_build              1.0
 PortGroup           cltversion                   1.0
 
-epoch               0
+epoch               1
```

Comment:
Why do you need to increase the epoch?

Reply:
it is how the automatic migration of libgcc works. see the activate hacks in the gcc12 Portfile.

Reply:
https://trac.macports.org/wiki/PortfileRecipes#deactivatehack